tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -33,97 +33,106 @@ public function testFix($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|string $expectedWithKeepMultipleSpacesAfterComma
+     *
+     * @dataProvider testFixProvider
+     */
+    public function testFixKeepMultipleSpacesAfterComma($expected, $input = null, $expectedWithKeepMultipleSpacesAfterComma = null)
```

Comment:
if you want to have one data provider, split parameters into `$exected, $input = null, array $configuration = null`